crest/Assets/Crest/Crest/Scripts/LodData/RegisterFlowInput.cs

```diff
@@ -21,7 +27,20 @@ public class RegisterFlowInput : RegisterLodDataInputDisplacementCorrection<LodD
 
         protected override string ShaderPrefix => "Crest/Inputs/Flow";
 
+        [Header("Spline settings")]
+        [SerializeField]
+        float _radius = 20f;
+        [SerializeField]
+        int _subdivisions = 1;
+        [SerializeField]
+        int _smoothingIterations = 0;
+        [SerializeField]
+        float _speed = 5f;
```

Comment:
The behaviour of this property is uncertain with the name _speed. It appears that this behaves as max speed from what I tested (unless it is negative because of Mathf.Min). Not sure how you want to handle this. We could have a base speed and a separate property to set the max speed instead?

Reply:
It is intended to be the actual flow speed at that point. There is a bug however where if the spline is closed things are broken which may be confusing the matter. If your spline is open, this value should set the flow speed at hte spline point.

I should probably rename it to velocity as speed should not be negative, i will do that.

Lmk if this makes sense.

Reply:
Sorry. It doesn't.

> It is intended to be the actual flow speed at that point.

By this you mean _flowSpeed on flow spline point (not _speed on the spline)?

From the shader (_speed * _flowSpeed if I understand correctly)
```shader
return wt * _Speed * input.speed * input.axis;
```

Here _speed is just a multiplier. So if no flow data components are on the spline point then this will ultimately be zero and there will be no flow.

From the other issue:
> So if the component is not attached to a spline point, the default flow will be used.

It sounds like _speed is default flow here?

Reply:
Ok I think multiplying two speeds is making a mess here.

I'm going to remove the speed param from the material.

Then the only speed control will be the velocity values on the spline which will have a non zero default, as I just added in the latest commits.

So if you only create a vanilla spline it will have some fixed default velocity, then if you click the individual spline points you can change the default velocity.

Reply:
Done. There is now only one flow speed control - the Flow Velocity setting on the spline points, which has a reasonable default value. Let me know if this is clear now.

Reply:
Works well.